src/libcore/ptr/const_ptr.rs

```diff
@@ -295,6 +295,64 @@ impl<T: ?Sized> *const T {
         intrinsics::ptr_offset_from(self, origin)
     }
 
+    /// Returns whether two pointers are guaranteed equal. Returns `true` whenever the
+    /// pointers are known to be equal.
```

Comment:
This intro still sounds rather odd, it is basically saying the same thing twice, or at least it sounds like it does.